tests/unit/src/kakarot/instructions/test_push_operations.cairo

```diff
@@ -16,441 +17,40 @@ from kakarot.instructions.push_operations import PushOperations
-    let result = PushOperations.exec_push27(ctx);
-    TestHelpers.assert_stack_last_element_contains_uint256(
-        result.stack, Uint256(0xFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF, 0xFFFFFFFFFFFFFFFFFFFFFF)
-    );
-
-    return ();
-}
-
-@external
-func test__exec_push28_should_add_28_byte_to_stack{
-    syscall_ptr: felt*, pedersen_ptr: HashBuiltin*, range_check_ptr, bitwise_ptr: BitwiseBuiltin*
-}() {
-    alloc_locals;
-    let ctx: model.ExecutionContext* = TestHelpers.init_context_with_bytecode(29, 0xFF);
-    let result = PushOperations.exec_push28(ctx);
-    TestHelpers.assert_stack_last_element_contains_uint256(
-        result.stack, Uint256(0xFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF, 0xFFFFFFFFFFFFFFFFFFFFFFFF)
-    );
-
-    return ();
-}
-
-@external
-func test__exec_push29_should_add_29_byte_to_stack{
-    syscall_ptr: felt*, pedersen_ptr: HashBuiltin*, range_check_ptr, bitwise_ptr: BitwiseBuiltin*
-}() {
-    alloc_locals;
-    let ctx: model.ExecutionContext* = TestHelpers.init_context_with_bytecode(30, 0xFF);
-    let result = PushOperations.exec_push29(ctx);
-    TestHelpers.assert_stack_last_element_contains_uint256(
-        result.stack, Uint256(0xFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF, 0xFFFFFFFFFFFFFFFFFFFFFFFFFF)
-    );
-
-    return ();
-}
-
-@external
-func test__exec_push30_should_add_30_byte_to_stack{
-    syscall_ptr: felt*, pedersen_ptr: HashBuiltin*, range_check_ptr, bitwise_ptr: BitwiseBuiltin*
-}() {
-    alloc_locals;
-    let ctx: model.ExecutionContext* = TestHelpers.init_context_with_bytecode(31, 0xFF);
-    let result = PushOperations.exec_push30(ctx);
-    TestHelpers.assert_stack_last_element_contains_uint256(
-        result.stack, Uint256(0xFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF, 0xFFFFFFFFFFFFFFFFFFFFFFFFFFFF)
-    );
-
-    return ();
-}
-
-@external
-func test__exec_push31_should_add_31_byte_to_stack{
-    syscall_ptr: felt*, pedersen_ptr: HashBuiltin*, range_check_ptr, bitwise_ptr: BitwiseBuiltin*
-}() {
-    alloc_locals;
-    let ctx: model.ExecutionContext* = TestHelpers.init_context_with_bytecode(32, 0xFF);
-    let result = PushOperations.exec_push31(ctx);
-    TestHelpers.assert_stack_last_element_contains_uint256(
-        result.stack, Uint256(0xFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF, 0xFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF)
-    );
-
-    return ();
-}
-
-@external
-func test__exec_push32_should_add_32_byte_to_stack{
-    syscall_ptr: felt*, pedersen_ptr: HashBuiltin*, range_check_ptr, bitwise_ptr: BitwiseBuiltin*
-}() {
-    alloc_locals;
-    let ctx: model.ExecutionContext* = TestHelpers.init_context_with_bytecode(33, 0xFF);
-    let result = PushOperations.exec_push32(ctx);
-    TestHelpers.assert_stack_last_element_contains_uint256(
-        result.stack,
-        Uint256(0xFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF, 0xFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF),
-    );
+    TestHelpers.assert_stack_last_element_contains_uint256(result.stack, Uint256(0xFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF,  res - 0x01));
 
     return ();
-}
+} 
```

Comment:
enable workflow in your repo to have the CI format your code
and run `make format` locally as well!